bower.json

```diff
@@ -9,9 +9,22 @@
     "testing"
   ],
   "devDependencies": {
-    "web-component-tester": "^4.0.0",
-    "webcomponentsjs": "webcomponents/webcomponentsjs#^0.7.0",
-    "webcomponentsjs-v1": "webcomponents/webcomponentsjs#v1"
+    "web-component-tester": "^4.0.0 || 6.0.0-prerelease.5",
+    "webcomponentsjs": "webcomponents/webcomponentsjs#^0.7.0 || v1"
```

Comment:
Let's pin it to the prerelease https://github.com/webcomponents/webcomponentsjs/releases/tag/v1.0.0-rc.1